doc/source/quickstart.rst

```diff
@@ -82,12 +85,11 @@ install most of these with::
     sudo apt-get update
     sudo apt-get install -y build-essential ccache git zlib1g-dev python2.7 python2.7-dev libncurses5:i386 libstdc++6:i386 zlib1g:i386 openjdk-8-jdk unzip ant ccache autoconf libtool
```

Comment:
Maybe we should update python version for debian/ubuntu:
```suggestion
    sudo apt-get install -y build-essential ccache git zlib1g-dev python3.7 python3.7-dev libncurses5:i386 libstdc++6:i386 zlib1g:i386 openjdk-8-jdk unzip ant ccache autoconf libtool
```
**Note:** I updated this line at #2105, but I will fix the merge conflicts, if you want to do it in here, no problem (I wil have to do it anyway)

Reply:
Done, I went with python3 and python3-dev meta package rather than explicit full version. I hope that's fine